app/templates/errors/410.html

```diff
@@ -0,0 +1,23 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Service no longer available - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="error-page">
+    <div class="index-page grid-row">
+        <div class="column-two-thirds">
+            <header class="page-heading-smaller">
+                <h1>Service no longer available</h1>
+            </header>
+            <p>
+                The requested service was on a framework that is no longer live and so it is no longer available through the Digital Marketplace.
```

Comment:
This message is specifically about gone services. I'm not sure it should be in a generic 410 template. Could we handle the 410 explicitly in the service show view if we need a specific service show message?